packages/shared/src/components/profile/ProfileForm.tsx

```diff
@@ -123,6 +131,28 @@ export default function ProfileForm({
       const filledFields = Object.keys(data).filter(
         (key) => data[key] !== undefined && data[key] !== null,
       );
+      if (shouldShowMyFeed) {
```

Comment:
Just a note here, I think we can remove this check.
There could be an edge case where the users AB test might have been removed/changed.

I think if they have local feed settings, we should just always apply them anyway.
What do you think?

Reply:
Very sharp thinking. Let me remove this one.